server/main.py

```diff
@@ -57,9 +57,9 @@ def interop_get(key):
     return interop.get_data(key)
 
 
-@app.route("/interop/telemetry")
+@app.route("/mav/telemetry")
 def interop_telemetry():
-    return jsonify(interop.telemetry_json)
+    return mav.get_telemetry()
```

Comment:
I feel like it's still a good idea to have both route: `/interop/telemetry` and `/mav/telemetry`. The interop telemetry route returns the Telemetry JSON Data last submitted to the Interop Server, and I feel like that is still important to know, at least for logging purposes. Consider adding a new route for this instead of editing the existing one.